doc/nrf/known_issues.rst

```diff
@@ -1365,6 +1365,18 @@ NCSDK-16644: :ref:`caf_sensor_manager` macro incorrectly converts float to senso
 
   **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``7e8c23a6632632f0cee885abe955e37a6942911d``).
 
+.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSIDB-925: Event listeners in :ref:`app_event_manager` may overlap when using non default naming convention.
+  Due to relaying on section sorting by the name and added "magic" postfixes, to locate the event listener start and stop address ranges,
+  using non default event naming scheme may break the expected listeners sorting.
+  In such situation one of the event listeners array may be actually placed inside the other.
+  This results in the execution of the event listeners also when the event that contains its listeners inside is processed.
```

Comment:
```suggestion
  Because of this, event listeners are executed even when they are placed inside the event. 
```

Reply:
please double-check if the meaning is correct

Reply:
The option created by @divipillai chosen.